lochness/rpms/__init__.py

```diff
@@ -110,22 +110,29 @@ def get_run_sheets_for_datatypes(target_df_loc: Union[Path, str]) -> None:
             raw_modality_path.mkdir(exist_ok=True, parents=True)
 
             if modality == 'surveys':
-                run_sheet_output = raw_modality_path / \
-                        f'{subject}.{study[:-2]}.Run_sheet_PennCNB.csv'
+                run_sheet_output_prefix = raw_modality_path / \
+                        f'{subject}.{study[:-2]}.Run_sheet_PennCNB'
             else:
-                run_sheet_output = raw_modality_path / \
-                        f'{subject}.{study[:-2]}.Run_sheet_{modality}.csv'
-
-            if run_sheet_output.is_file():
-                target_df = pd.read_csv(target_df_loc)
-                same_df = pd.read_csv(run_sheet_output).equals(target_df)
-                if same_df:  # skip everything if it is already there
-                    return
+                run_sheet_output_prefix = raw_modality_path / \
+                        f'{subject}.{study[:-2]}.Run_sheet_{modality}'
+
+            # convert month to datapoint
+            if modality in ['eeg', 'mri']:
+                # two month : first timepoint
+                # four month : second timpeoint
+                time_to_timepoint = {2: 1, 4: 2}
+            elif fieldname == 'PennCNB':
+                time_to_timepoint = {2: 1, 4: 2, 8: 3, 14: 4, 16: 5}
+            else:  # actigraphy & EMA
+                time_to_timepoint = dict(zip(range(2, 15), range(1, 14)))
```

Comment:
There are many other forms that are assigned to multiple time points:
![image](https://user-images.githubusercontent.com/35086881/169564383-ca0ca579-9956-4406-9992-d15bcedbe09e.png)

Can you confirm that the comment `# actigraphy & EMA` is underinclusive?

Reply:
It is according to the table that Larry shared in the email. We may have to change this in future PR, but it works fine for now.